src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Base64Decoder.cs

```diff
@@ -479,17 +749,20 @@ private static unsafe void Avx2Decode(ref byte* srcBytes, ref byte* destBytes, b
             destBytes = dest;
         }
 
+        // TODO https://github.com/dotnet/runtime/issues/63331: Replace this once that's available.
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private static Vector128<byte> SimdShuffle(Vector128<byte> left, Vector128<byte> right, Vector128<byte> mask8F)
         {
             Debug.Assert((Ssse3.IsSupported || AdvSimd.Arm64.IsSupported) && BitConverter.IsLittleEndian);
 
-            if (AdvSimd.Arm64.IsSupported)
+            if (Ssse3.IsSupported)
             {
-                right &= mask8F;
+                return Ssse3.Shuffle(left, right);
+            }
+            else
+            {
+                return AdvSimd.Arm64.VectorTableLookup(left, Vector128.BitwiseAnd(right, mask8F));
             }
-
-            return Vector128.ShuffleUnsafe(left, right);
         }
```

Comment:
This looks like it's unnecessarily reverting changes from #80963 to use `ShuffleUnsafe`.

Reply:
Yup, bad merge.